lib/interface_quda.cpp

```diff
@@ -289,28 +267,28 @@ static void profilerStart(const char *f) {
   static int target_count = 0;
   static unsigned int i = 0;
   if (do_not_profile_quda){
-    cudaProfilerStop();
+    device::profile::stop();
```

Comment:
I'm a little confused about why this logic exists in the first place... if we're not profiling QUDA, why do we need to "stop" profiling QUDA in `profilerStart`? I could see a world where this maybe lets users update `QUDA_DO_NOT_PROFILE` from the environment in the middle of a run, but I still don't see how the logic of "stopping" when you're requesting a start makes sense.